api/envoy/config/metrics/v3/metrics_service.proto

```diff
@@ -55,4 +68,7 @@ message MetricsServiceConfig {
   // and the tag extracted name will be used instead of the full name, which may contain values used by the tag
   // extractor or additional tags added during stats creation.
   bool emit_tags_as_labels = 4;
+
+  // Specify which metrics types to emit for histograms. Defaults to SUMMARY_AND_HISTOGRAM.
+  HistogramMetrics histogram_metrics = 5 [(validate.rules).enum = {defined_only: true}];
```

Comment:
nit:` histogram_emit_mode` may be better name?

Reply:
sure

Reply:
@ramaraochavali done